docroot/profiles/custom/sitenow/js/sitenow_detector.behaviors.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file
+ * Chosen.
+ */
+(function ($, Drupal) {
+    Drupal.behaviors.sitenowDetector = {
+      attach: function (context) {
+        $('meta[content="SiteNow v2 (https://sitenow.uiowa.edu)"]', context).once('sitenowDetector').each(function () {
```

Comment:
My only concern is that if we ever remove that metatag, possibly unintentionally, this will stop working. Could we just attach it to the document?

Reply:
yes of course, my mind must have gotten stuck in needing the metatag for some reason. I'll change that.